pkg/mover/chart.go

```diff
@@ -256,12 +257,68 @@ func (cm *ChartMover) loadChartFromIntermediateBundle(bundlePath string) error {
 // loadChartFromPath load the chart in memory from a given path
 func (cm *ChartMover) loadChartFromPath(path string) error {
 	var err error
-	if cm.chart, err = loader.Load(path); err != nil {
-		return &ChartLoadingError{Path: path, Inner: err}
+	dir := path
+	if isChartFile(path) {
```

Comment:
well, this method is not saying that's a chart tarball, it's saying that's not a directory right?

Reply:
to be picky yes, it could be a link or something else... but then the file extract will fail it is not a tarball

Reply:
changed to `if !isChartDir(path) {`